tests/bluetooth/bluetooth/prj.conf

```diff
@@ -1,6 +1,6 @@
 CONFIG_BT=y
 CONFIG_BT_CTLR=n
-CONFIG_BT_NO_DRIVER=y
+CONFIG_BT_H4=n
```

Comment:
It may not be obvious that this effectively disables all BT drivers. Perhaps a comment?

Reply:
Well, it disables two drivers, not all of them. However, these are sufficient considering the board targets that the bluetooth unit tests are built against.

Reply:
> Well, it disables two drivers, not all of them. 

It's all of them for this specific project, isn't it?